sites/backend/scripts/prebuild-i18n.mjs

```diff
@@ -6,7 +6,7 @@ import yaml from 'js-yaml'
 /*
  * List of supported languages
  */
-const locales = ['en', 'es', 'de', 'fr', 'nl']
+// const locales = ['en', 'es', 'de', 'fr', 'nl']
```

Comment:
I was trying to reduce the number of places where we have to manually add languages. I believe I did this because with this hard coded array, Ukrainian is not being prebuilt and then the site build fails, but I can double check